router/types/forward.go

```diff
@@ -86,3 +89,28 @@ func (o JSONObject) MarshalJSON() ([]byte, error) {
 	// primitive, return raw bytes.
 	return o.primitive, nil
 }
+
+func (d Duration) MarshalJSON() ([]byte, error) {
+	return json.Marshal(time.Duration(d).String())
```

Comment:
We should marshal to the nanoseconds value so it's backwards compatible with older PFM